### PR TITLE
Removes numerical scale

### PR DESCRIPTION
**Pull request by tkohr**
PR removes the numerical scale, since the final value for this will depend on the DPI in which the generated PNG will be printed.

### CODE REVIEW

File: src/printer/scalebar.js [hunk 1]
```diff
@@ -125,27 +123,19 @@ function getScaleBarParams(frameState, spec) {
...
 
   const scaleText = `${scaleNumber} ${scaleUnit}`;
   const scaleTextWidth = ctx.measureText(scaleText).width;
 
-  const scaleTitle = (spec.scaleBar.template
-    ? spec.scaleBar.template
-    : DEFAULT_TITLE
-  ).replace('{mapScale}', mapScale);
-  const scaleTitleWidth = ctx.measureText(scaleTitle).width;
-
```
Comment (LHBruneton-C2C) [inline src/printer/scalebar.js:142]: Just saying, 'template' was never described as part of the ScaleBarSpec in the README.

**Review by LHBruneton-C2C — APPROVED** [review]
LGTM ! Not tested though.

File: src/printer/scalebar.js (hunk 1 above)
Comment (tkohr) [reply src/printer/scalebar.js:142]: Oups, you're right, another reason to get rid of it ;-)

**Comment by tkohr** [comment]
Thanks @LHBruneton-C2C

**Pull request merged**